src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs

```diff
@@ -106,7 +107,7 @@ public void WriteToReadOnlyFile()
             try
             {
                 // Operation succeeds when being run by the Unix superuser
-                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && geteuid() == 0)
+                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && !RuntimeInformation.IsOSPlatform(OSPlatform.Browser)  && geteuid() == 0)
```

Comment:
```suggestion
                if (PlatformDetection.IsSuperUser)
```

Reply:
(multiple places)

Reply:
Thanks for the IsSuperUser hint.  Did not know about that.

Reply:
Addressed in recent commits